core-bundle/src/Resources/contao/models/FilesModel.php

```diff
@@ -21,21 +21,21 @@
  * The files themselves reside in the files directory. This class only handles
  * the corresponding database entries (database aided file system).
  *
- * @property string|integer      $id
- * @property string|integer|null $pid
- * @property string|integer      $tstamp
- * @property string|null         $uuid
- * @property string              $type
- * @property string              $path
- * @property string              $extension
- * @property string              $hash
- * @property string|boolean      $found
- * @property string              $name
- * @property string|float        $importantPartX
- * @property string|float        $importantPartY
- * @property string|float        $importantPartWidth
- * @property string|float        $importantPartHeight
- * @property string|array|null   $meta
+ * @property integer           $id
+ * @property integer|null      $pid
```

Comment:
This seems to have been a previous mistake. `tl_files.pid` is always an UUID or NULL. So this should always have been `string|null` (just like `tl_files.uuid`).